packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -26,7 +27,7 @@ process.env.TOKEN_SECRET = randomString();
 // import the express app after setting the env variables
 const { app } = require('../../../app');
 
-const esIndex = randomString();
+const esIndex = 'localrun-es';
```

Comment:
Why does it need to be this value for unit tests? Wouldn't it be preferable for these tests to use their own index to ensure isolation?

Reply:
Yes. I'll be putting that back.